recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
```

Comment:
We had a discussion in our general meeting recently and agreed to always prefix Python packages with `python-`. Given that `conda` is a language agnostic packaging tool and new packages from other languages are being added/proposed regularly, it makes sense to us to do the prefixing to keep track of what language this relates to. Once that is addressed, we will be happy to merge this nice recipe.